tools/parallel-lint.sh

```diff
@@ -27,6 +27,10 @@ if $ALL; then
 	if php -r 'exit( PHP_VERSION_ID < 70000 ? 0 : 1 );'; then
 		SKIPS+=( -o -name php7 )
 	fi
+	if php -r 'exit( PHP_VERSION_ID < 70200 ? 0 : 1 )'; then
+      # Plugin requires PHP 7.2 or later.
+      SKIPS+=( -o -path ./projects/plugins/crm )
+  fi
```

Comment:
Indentation is weird.
```suggestion
	if php -r 'exit( PHP_VERSION_ID < 70200 ? 0 : 1 )'; then
		# Plugin requires PHP 7.2 or later.
		SKIPS+=( -o -path ./projects/plugins/crm )
	fi
```

Reply:
Thank you! For some reason, my IDE decide to use spaces instead of tabs.